### DISCUSSION TITLE
Precise hexagonal pixel modeling and an easy-sharing storage scheme for remote sensing images based on discrete global grid system

### DISCUSSION THREAD

**Discussion by ajfriend** [uber/h3#876] [H3 Spotlight]
Precise hexagonal pixel modeling and an easy-sharing storage scheme for remote sensing images based on discrete global grid system
Qishuang Liang , Jianbin Zhou , Jin Ben , Yihang Chen , Xinhai Huang , Junjie Ding and Jinchi Dai
Institute of Surveying and Mapping, PLA Strategic Support Force Information Engineering University, Zhengzhou, People’s Republic of China

[Precise_hexagonal_pixel_modeling_and_an_easy-shari.pdf](https://github.com/user-attachments/files/16215441/Precise_hexagonal_pixel_modeling_and_an_easy-shari.pdf)

**Comment by dfellis** [comment]
This one doesn't seem to be related to H3. There is only a reference at the end to it that doesn't appear to actually be cited in the text. They have created their own aperture 4 hex grid.
